src/batch/src/task/task_manager.rs

```diff
@@ -55,7 +55,7 @@ impl BatchManager {
         let task_id = task.get_task_id().clone();
         let task = Arc::new(task);
 
-        task.clone().async_execute()?;
+        task.clone().async_execute().await?;
```

Comment:
If we await here, the `fire_task` call becomes: fire task and wait for it to complete. But `fire_task` should mean fire task and let it run in backend.

Reply:
`fire_task` will wait for the completion of building executors.
The real execution is in another async task spawned below.

The extra latency added is the initialization of Hummock's iterator according to the metadata already pinned in `LocalVersionManager`. Since we prioritize caching metadata over SST, we can assume most of the time that the metadata to be used has already been cached in the local memory.

Since a batch query will be issued only after a certain epoch is committed. There is no blocking on the path of iterator creation due to the epoch waiting in Hummock.

Without doing so, I have to initialize some Frontend Service and Frontend Client for the message of `successful creation of iterators` to be passed from CN to the Frontend. At this point, this PR is the only demand to do so.

If there are indeed many types of such messages from CN to Frontend, I will change to allocate another communication path.